LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_ThermoCouple.TcPOU

```diff
@@ -3,7 +3,7 @@
   <POU Name="FB_ThermoCouple" Id="{cc00895a-0264-4382-820e-9cfa9062e545}" SpecialFunc="None">
     <Declaration><![CDATA[FUNCTION_BLOCK FB_ThermoCouple
 (*
-	Handles scaling and default diagnostics for thermocouple terminals
+	Deprecated as of 2020-03-02, please use FB_TempSensor instead
```

Comment:
I recommend including a message pragma to warn about deprecation:
https://infosys.beckhoff.com/content/1033/tc3_plc_intro/18014401039043723.html?id=4216455307877710813